sdk/dns/Azure.ResourceManager.Dns/src/autorest.md

```diff
@@ -3,9 +3,47 @@
 Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
-
 azure-arm: true
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e0996d4f6dbca40ebf2fa4abf9a1cba45ada94d8/specification/dns/resource-manager/readme.md
- 
+library-name: Dns
+namespace: Azure.ResourceManager.Dns
+require: https://github.com/Azure/azure-rest-api-specs/blob/48a49f06399fbdf21f17406b5042f96a5d573bf0/specification/dns/resource-manager/readme.md
+clear-output-folder: true
+skip-csproj: true
+modelerfour:
+  lenient-model-deduplication: true
+
+# temp
+list-exception:
+- /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/dnsZones/{zoneName}/{recordType}/{relativeRecordSetName}
```

Comment:
I believe in our latest implementation of autorest.csharp, this line can be removed.